code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -42,11 +42,33 @@
 	desc = "Jeez. I hope that's not for lunch."
 	gender = PLURAL
 	density = 0
-	anchored = 1
-	luminosity = 1
+	anchored = TRUE
 	icon = 'icons/effects/effects.dmi'
 	icon_state = "greenglow"
 
+/obj/effect/decal/cleanable/greenglow/New()
+	..()
```

Comment:
Prefer `. = ..()` as it is slightly less likely to fuck up in the future but not strictly needed.

Reply:
I can't say I've seen it cause issues before nor have I heard of it causing issues. I'm open to changing it if you can give me a bit more than personal taste.

Reply:
what . = ..() means it that it sets the return value to the return value of the parent, it's useful if you want to for example do something like

```
. = ..()
CODE SPECIFIC TO THIS CHILD WHICH NEEDS TO OPERATE AFTER THE PARENT'S CODE IS EXECUTED
return .
```
Allows you to return ..() without immediately ending the proc. In short, it's not necessary if you aren't returning anything like in a New()